### DISCUSSION TITLE
Make the buttons react when clicked

### DISCUSSION THREAD

**Discussion by AaronBangs** [Franciscan-CS-Club/fix-franciscan-ordering#22] [Implemented ideas] [answered]
At present it is very difficult to tell when you have actually clicked a button. It would be a nice QOL feature if the buttons shrank slightly or changed color when clicked, just so you know when you've clicked it.

**Comment by 2br-2b** [comment]
Check out #23, which adds a css page.

Editing style.css will add css to the webpage.

**Comment by TJJP** [comment, marked as answer]
Fixed and added with 1d4ef93b434acf4fd3e14b2f23c8d186adff20f9
